app/assets/css/context-aware-editor/_index.scss

```diff
@@ -163,8 +163,8 @@
 
 // When toolbar is disabled, ensure proper spacing
 .app-context-aware-editor--wrapper:not(
-    .app-context-aware-editor--with-toolbar
-) {
+        .app-context-aware-editor--with-toolbar
+    ) {
```

Comment:
Not sure how these snuck in here but it doesn't look relevant to this change and isn't what the formatter wants things to do - could you check this file out of main?